<launch>
    <include file="$(find pylon_camera)/launch/pylon_camera_node.launch">
        <arg name="startup_user_set" value="1" />
    </include>

    <include file="$(find agv_prbt_2022)/launch/agv_prbt.launch"/>

    <node ns="pylon_camera_node" name="image_rectify" pkg="image_proc" type="image_proc" />
    <node name="image_processing" pkg="agv_prbt_2022" type="image_processing_node.py" />

    <node name="agv_prbt_application" pkg="agv_prbt_2022" type="agv_prbt_application.py" output="screen" />
    
</launch>